draft-ietf-quic-transport.md

```diff
@@ -1311,6 +1339,10 @@ Version Negotiation packet.
 A client MUST ignore a Version Negotiation packet that lists the client's chosen
 version.
 
+A client MAY attempt 0-RTT after receiving a Version Negotiation packet.  A
+client that sends additional 0-RTT packets MUST NOT reset the packet number to 0
+after a Retry packet, see {{retry-0rtt-pn}}.
```

Comment:
Since this section is about Version Negotiation packets, the mention of Retry packets seems a *non sequitur*.  It's fine to reset the 0-RTT packet number space after a VN packet, because the previous 0-RTT packets will be of a version unknown to the server and therefore will just be discarded (or rather, will trigger more VN packets).

Reply:
It's not OK to reset the packet numbers, because you just created a two-time pad.  The same key and nonce, with different inputs (the version number).  That's disastrous.

The mention of Retry is a mistake though.